lib/compat/wordpress-6.1/class-gutenberg-rest-edit-site-export-controller.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * REST API: WP_REST_Edit_Site_Export_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ */
+
+/**
+ * Controller which provides REST endpoints for exporting current templates
+ * and template parts either as a zip bundle or as edits to the current enabled
+ * theme's files.
+ *
+ * @since 5.9.0
+ *
+ * @see WP_REST_Controller
+ */
+class Gutenberg_REST_Edit_Site_Export_Controller_6_1 extends Gutenberg_REST_Edit_Site_Export_Controller {
+
+
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/export_to_theme_files',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'export_to_theme_files' ),
+					'permission_callback' => array( $this, 'permissions_check' ),
```

Comment:
Should we change the capability of the Gutenberg_REST_Edit_Site_Export_Controller class to check for `export`?

Reply:
We could add an export cap! But ... given how this is only available with that constant IDK.

Reply:
I meant, should we change the permissions for the endpoint from `edit_theme_options` to `export`.